cicecore/cicedyn/general/ice_flux.F90

```diff
@@ -1231,7 +1231,10 @@ subroutine scale_fluxes (nx_block, ny_block, &
             fsens   (i,j) = fsens   (i,j) * ar
             flat    (i,j) = flat    (i,j) * ar
             fswabs  (i,j) = fswabs  (i,j) * ar
+            if (flwout(i,j) > -puny) & 
+               flwout  (i,j) = -stefan_boltzmann *(Tf(i,j) + Tffresh)**4
             flwout  (i,j) = flwout  (i,j) * ar
+            ! Special case where aice_init was zero and aice > 0.
```

Comment:
why is the comment 2 lines below the if statement?

Reply:
Whoops. Fixed now.